docs/Documentation/Software_Tools/spack.md

```diff
@@ -69,16 +69,16 @@ The environment variable `SPACK_SHORT_SPEC` in `build.env` contains the Spack co
 The 7-character short hash is also included, and should be excluded from any spack install command.
 
 
-|symbols |description|
-|----|-----|
+|symbols       | description |
```

Comment:
Suggestion to make this table a little cleaner, prevent the word wrap issue on target=value:

| <div style="width:98px">symbols</div>  |  description  |

I dislike doing this, but markdown is not playing nice with the table formatting, since the character length of "symbols" is shorter than "value=target". Other idea: change the header title to something like "symbol/shortcut", or any single word longer than "value=target".

Reply:
Hi Tim, I've added the line you suggested but It seems like Md only does what it wants. (The table size didn't change)